webapp/src/components/admin_settings/job_table/job_table.tsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import {DateRange} from 'react-day-picker';
+
+import {Job} from '../types/types';
+
+import {craeteJob, getJobs} from './actions';
+
+import JobDateTime from './job_date_time';
+import JobDownloadLink from './job_download_link';
+import JobScheduleModal from './job_schedule_modal';
+import './job_schedule_modal.scss';
+
+export type Props = {
+    jobs: Job[];
+    showModal: boolean;
+    className?: string;
+}
+
+type State = {
+    jobs: Job[];
+    showModal: boolean;
+    className?: string;
+}
+
+export default class JobTable extends React.Component<State, Props> {
```

Comment:
Is there a reason this is a class component? (Just surprised to see it after all the functional components above)